src/types/Todo.ts

```diff
@@ -0,0 +1,6 @@
+import { TodoWithoutUser } from './TodoWithoutUser';
```

Comment:
Consider adding a comment or description to explain the purpose of the import. 📝